crates/starknet/src/constants.rs

```diff
@@ -2,14 +2,20 @@ use std::net::{IpAddr, Ipv4Addr};
 
 use starknet_types::chain_id::ChainId;
 
-pub(crate) const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
+pub const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
     env!("CARGO_MANIFEST_DIR"),
     "/accounts_artifacts/OpenZeppelin/0.5.1/Account.cairo/Account.json"
```

Comment:
This is from one of the first PRs, but why not rename this file to Account.casm

Reply:
cairo0 artifacts never used the .casm extension, it was always .json

Reply:
I remember having a similar discussion although not for the same extensions. https://github.com/0xSpaceShard/starknet-devnet-rs/pull/122#discussion_r1285565407

Reply:
Renamed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/9795932d96e5df85b13acc40259ea158d5718038